sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonHighlighter.java

```diff
@@ -106,7 +106,7 @@ public void init() {
   public void visitFile(@Nullable AstNode astNode) {
     docStringTokens = new HashSet<>();
     newHighlighting = context.newHighlighting();
-    InputFile inputFile = context.fileSystem().inputFile(context.fileSystem().predicates().is(getContext().getFile().getAbsoluteFile()));
+    InputFile inputFile = context.fileSystem().inputFile(context.fileSystem().predicates().is(getContext().pythonFile().file().getAbsoluteFile()));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "file"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)